src/utilities/prTitleParser.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { parsePRTitle } from './prTitleParser';
+
+var title: string = '';
+
+describe('parsePRTitle', (): void => {
+  describe('given containing MAJOR bump', (): void => {
+    beforeEach((): void => {
+      title = 'bump @types/jest from 26.0.12 to 27.0.13';
+    });
```

Comment:
I would just write this:
```suggestion
    const title = 'bump @types/jest from 26.0.12 to 27.0.13';
```

Reply:
Should the title have a 'type' ? Like `chore: bump ...`